py/embedded-server/README_PyPi.md

```diff
@@ -0,0 +1,41 @@
+# Deephaven Python Embedded Server
+
+Embeds the Deephaven Core server into a python module, so that it can be started from python, with tables produced
+directly as part of startup.
```

Comment:
I think this should be a more python-end-user introduction - we shouldn't be focused on the "embeds" part of it. A link to deephaven-core is fine.